website/static/playground.js

```diff
@@ -439,6 +439,25 @@ function getExample(parser) {
         "  }",
         "}"
       ].join("\n");
+    case "json":
+      // Excerpted & adapted from Wikipedia, under the Creative Commons Attribution-ShareAlike License
+      // https://en.wikipedia.org/wiki/JSON#Example
+      // https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License
```

Comment:
I think one link is enough sine the license is mentioned already in the comment above